nexus/reconfigurator/planning/src/blueprint_builder/builder.rs

```diff
@@ -1081,6 +1178,33 @@ pub mod test {
                 );
             }
         }
+
+        // On any given zpool, we should have at most one zone of any given
```

Comment:
Is this function actually used anywhere?

Reply:
Yeah, most/all of the tests in this file and `planner.rs` call this after generating blueprints to do basic sanity checking